crates/bevy_scene/src/scene_loader.rs

```diff
@@ -9,6 +9,11 @@ use bevy_utils::BoxedFuture;
 #[cfg(feature = "serialize")]
 use serde::de::DeserializeSeed;
 
+/// Asset loader for a Bevy dynamic scene (`.scn` / `.scn.ron`).
+///
+/// The loader handles assets serialized with [`DynamicScene::serialize()`].
+///
+/// [`DynamicScene::serialize()`]: crate::DynamicScene::serialize
```

Comment:
Nit: I think this will auto-link since it's in scope
```suggestion
/// The loader handles assets serialized with [`DynamicScene::serialize`].
```

Reply:
Ah yes sorry, I have the habit of linking without the type name prefix, which doesn't auto-link. I will fix.